app/components/summary_card/view.html.erb

```diff
@@ -0,0 +1,10 @@
+<section class="app-summary-card govuk-!-margin-bottom-6">
+  <header class="app-summary-card__header">
+    <h<%= @heading_level %> class="app-summary-card__title">
```

Comment:
Just as a standard pattern, would it make sense to use accessor methods here instead of instance variables? i.e. `@heading_level` -> `heading_level`?

Reply:
fixed